src/components/Footer.svelte

```diff
@@ -1,5 +1,8 @@
 <script>
   import MozillaLogo from "./icons/MozillaLogo.svelte";
+
+  let rev = "__GLEAN_VERSION__";
```

Comment:
```suggestion
  const rev = "__REVISION__";
```

Reply:
@wlach I did eliminate this rev variable constantly. What do you think?

Reply:
Yeah, I was on the fence about it too-- in the end I think it is better to keep it, as it makes the logic below a little more understandable. Can you make that change? After that, I'd be happy to merge this. Pre-emptively approving.